src/lib/Security/Authentication/RedirectToDashboardListener.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\AdminUi\Security\Authentication;
+
+use Ibexa\AdminUi\Specification\SiteAccess\IsAdmin;
+use Ibexa\Core\MVC\Symfony\Security\Authentication\DetermineTargetUrlEvent;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\Security\Http\Util\TargetPathTrait;
+
+class RedirectToDashboardListener implements EventSubscriberInterface
+{
+    use TargetPathTrait;
+
+    /** @var array */
+    private array $siteAccessGroups;
+
+    public function __construct(
+        array $siteAccessGroups
+    ) {
+        $this->siteAccessGroups = $siteAccessGroups;
+    }
+
+    public static function getSubscribedEvents(): array
```

Comment:
Same as above.

Reply:
It is described in interface doc tho, not sure we need to duplicate this.